react/form/public/index.html

```diff
@@ -14,6 +14,13 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React Form Widget</title>
+    <script>
```

Comment:
is this simulating what the login widget should do?

Reply:
the server/backend should append the Keyclock server details for the client